projects/timer_fw102/src/main.c

```diff
@@ -0,0 +1,46 @@
+#include <stdbool.h>
+#include "gpio.h"
```

Comment:
Nitpick: the coding standards say there should be a blank line between <standard> includes and "midsun-specific" includes.